packages/core/src/fhir-deduplication/resources/condition.ts

```diff
@@ -53,7 +53,8 @@ export function groupSameConditions(conditions: Condition[]): {
         coding.system?.includes(SNOMED_CODE) ||
         coding.system?.includes(SNOMED_OID) ||
         coding.system?.includes(ICD_10_CODE) ||
-        coding.system?.includes(ICD_10_OID)
+        coding.system?.includes(ICD_10_OID) ||
+        coding.system?.includes(ICD_9_CODE)
```

Comment:
we should store `coding.system` in a const with the `.toLowerCase()` before the comparisons

Reply:
and we should do that in other places too. This is a pretty crucial step since we're probably filtering out codes that we want to keep

Reply:
its being done in the other places I just checked. Condition is the place it was missing